rotkehlchen/data_migrations/migrations/migration_10.py

```diff
@@ -25,7 +25,7 @@ def data_migration_10(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgress
     with rotki.data.db.conn.read_ctx() as cursor:
         accounts = rotki.data.db.get_blockchain_accounts(cursor)
     # steps are: ethereum accounts + potentially write to db + updating spam assets
-    progress_handler.set_total_steps(len(accounts.eth) + 2)
+    progress_handler.set_total_steps(len(accounts.eth) + 3)
 
```

Comment:
what is the extra one? you have in the comment only two

Reply:
Yes it's 2 + 1. It had off by one error. Without it you get this:

![2023-06-23_14-51](https://github.com/rotki/rotki/assets/1658405/22d00437-d2fe-4076-8762-d88dffd42b18)